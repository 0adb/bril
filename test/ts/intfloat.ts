var value = 1n;
// Floats are unsupported at the moment
// Also there seems to be no way for the TS parser to interpret BigInt operations as such, so that's out for now