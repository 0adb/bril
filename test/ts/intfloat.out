main {
  v0: int = const 1;
  value: int = id v0;
}
